src/Makefile.am

```diff
@@ -213,6 +213,7 @@ libprotobuf_lite_la_SOURCES =                                  \
   google/protobuf/repeated_field.cc                            \
   google/protobuf/wire_format_lite.cc                          \
   google/protobuf/io/coded_stream.cc                           \
+  google/protobuf/io/coded_stream_inl.h                        \
```

Comment:
This line needs to be removed. Please double check it's removed when you do up-integration.

Reply:
Done